src/components/InputDropdown/index.js

```diff
@@ -45,7 +45,6 @@ function InputDropdown({
         css={[styles.select, value === DEFAULT && styles.selectDefaultState]}
         onChange={handleChange}
         name={name}
-        defaultValue={DEFAULT}
```

Comment:
why was this default removed?

Reply:
See `defaultValue` comment in branch that this is being merged into:
https://github.com/helpsupply/helpsupply/pull/173#issue-402786967